src/org/omegat/filters2/master/PluginUtils.java

```diff
@@ -194,12 +194,10 @@ public static void loadPlugins(final Map<String, String> params) {
                 try (InputStream in = mu.openStream()) {
                     Manifest m = new Manifest(in);
                     if ("org.omegat.Main".equals(m.getMainAttributes().getValue("Main-Class"))) {
-                        // found main manifest - not in development mode
+                        // found a main manifest - not in development mode
                         foundMain = true;
-                        loadFromManifest(m, pluginsClassLoader, null);
```

Comment:
Here is a place we can understand `getUrl()` in `PluginInstaller` can be NULL because we pass `null` here.